draft-ietf-webpush-vapid.md

```diff
@@ -333,8 +333,8 @@ a public key.
 ## Using Restricted Subscriptions
 
 When a push subscription has been associated with an application server, the
-request for push message delivery MUST include proof of possession for the
-associated private key that was used when creating the push subscription.
+request for push message delivery MUST include a JWT signed by the private key
+that was used when creating the push subscription.
```

Comment:
Is it correct to refer to the _private key_ that was used to create the subscription, when that is done using a _public key_ that was derived from the private key?

What about:

> When a push subscription has been **restricted to** an application server, the
> request for push message delivery MUST include a **valid** JWT signed by the
> private key **paired with the public key used when creating the push subscription**.